stories/topics/proc-gcp-backup-container-image.adoc

```diff
@@ -3,7 +3,7 @@
 = Pulling the ansible-on-clouds-ops container image
 
 .Procedure
-* Pull the docker image for the `ansible-on-clouds-ops` container with the same tag as the foundation deployment.
+* Pull the docker image for the `ansible-on-clouds-ops` container with the same tag as the foundation deployment. If you are unsure of the version you have deployed, see xref:ref-aap-using-playbooks[Command Generator] and playbook *gcp_get_aoc_version* for more information on figuring out the current version of Ansible on Clouds deployment.
```

Comment:
New line before the second sentence, and

for more information on finding the current....

Reply:
fixed